### PR TITLE
Modal

### PR DESCRIPTION
**Pull request by gadgetfather**
I have created a modal. Please review it
Preview link : https://zjmfm.csb.app/modal/modal.html
image
image

### CODE REVIEW

**Review by Kevin-Solomon — APPROVED** [review]

**Review by RahullRawat — APPROVED** [review]

File: modal/modal.css [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    margin: auto;
+    padding: 20px;
+    width: 30%;
+  }
+  
+  
+/* The Close Button */
+.close {
+    color: #aaaaaa;
+    float: right;
```
Comment (jainPrajwal) [inline modal/modal.css]: Suggestion: Try not using float if you can with flex.

File: modal/modal.css [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+.modal-container{
+    display: none; 
+    position: fixed; 
+    z-index: 1; 
+    left: 0;
+    top: 0;
+    width: 100%; 
+    height: 100%; 
+    overflow: auto; 
+    background-color: rgb(0,0,0);
```
Comment (jainPrajwal) [inline modal/modal.css]: You dont require because you are cascading the background-color in next line.

**Review by jainPrajwal — COMMENTED** [review]
Added suggestions

File: modal/modal.css (hunk 2 above)
Comment (gadgetfather) [reply modal/modal.css]: oke i wrote that as a fallback value

File: modal/modal.css (hunk 1 above)
Comment (gadgetfather) [reply modal/modal.css]: yea i have removed it.

**Pull request merged**